### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by godcrampy**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [x] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
N/A

### Motivation / Use-Case
N/A

### Breaking Changes
Nope

### CODE REVIEW

**Comment by godcrampy** [comment]
No response for two weeks. Maybe this was a silly PR. Hence closing this!

**Comment by evilebottnawi** [comment]
@godcrampy We will merge it before v4 release to avoid conflicts unnecessary conflicts with v3

**Comment by godcrampy** [comment]
@evilebottnawi Thanks for the update, reopening PR. Sorry for the mess up.

**Comment by GChuf** [comment]
@godcrampy can your fix another typo?
Contributions: line 13: "this leads"

**Comment by evilebottnawi** [comment]
@godcrampy can you change branch on v4?

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ needed modifications to the code for the module. Because of that, we ask users
 with general support, "how-to", or "why isn't this working" questions to try one
 of the other support channels that are available.
 
-Your first-stop-shop for support for webpack-dev-server should by the excellent
+Your first-stop-shop for support for webpack-dev-server should start by the excellent
```
Comment (murtraja) [inline README.md:112]: should "be" the excellent
instead of
should "by" the excellent

But "start by" sounds good too!

**Pull request closed without merging**